spec/jobs/batch_item_processing_job_spec.rb

```diff
@@ -16,21 +16,21 @@
     allow(ingester).to receive(:ingest).and_return(work)
   end
 
-  describe '#perform' do
+  describe '#perform (via #perform_now)' do
     let!(:batch) { FactoryBot.create(:enqueued_batch, batch_items: [batch_item]) }
 
     it 'runs the ingester' do
-      job.perform(batch_item)
+      job.perform_now
```

Comment:
How does `perform_now` work without passing the bath_item reference?

Reply:
@bkeese `perform_now` (like `perform_later`) takes no arguments. Instead, it makes the arguments passed to the contructor available within the job object via the `#arguments` method. Using `#perform` directly does not run the callbacks, which is what we need to do here.

Reply:
@bkeese Actually... ☝️ this explanation is sort of wrong -- the _class_ method `#perform_now` _does_ take an argument, but the _instance_ method does not.

I was getting confused, because I was continuing to use the instance of the job but was then getting `ArgumentError: wrong number of arguments (given 1, expected 0)` whenever I called `#perform_now(batch_item)`.

Calling the instance method `#perform_now` without arguments works the same way (I think) as calling the class method `.perform_now` _with_ arguments... however, I think I'll submit a follow-up PR to change it to the latter, since that's the conventional way of doing it according to the Rails docs.